rockspec/api7-snowflake-2.0-1.rockspec

```diff
@@ -1,16 +1,16 @@
-package = "snowflake"
```

Comment:
We can leave this file unchanged as now we start from v2.1

Reply:
Can I publish to luarocks without changing the version number?

Reply:
Is it possible to publish to luarocks after only changing the version number?

Reply:
Yes.